.github/workflows/db-update-dev.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       PROJECT_ID: ${{ vars.DEV_MOBILITY_FEEDS_PROJECT_ID }}
       DB_NAME: ${{ vars.DEV_POSTGRE_SQL_DB_NAME }}
       REGION: ${{ vars.DEV_MOBILITY_FEEDS_REGION }}
-      RUN_POPULATE_SCRIPT: ${{ github.event.inputs.RUN_POPULATE_SCRIPT || false }}
+      RUN_POPULATE_SCRIPT: ${{ github.event.inputs.RUN_POPULATE_SCRIPT || 'true' }}
```

Comment:
It's the only one you changed (false to true). Is it intentional?

Reply:
Intentional, as it's the dev environment.